api/backend/ecs/utilities.go

```diff
@@ -1,14 +1,15 @@
 package ecsbackend
 
 import (
+	"strings"
```

Comment:
[nit] there are two auto-formatters for Go code: `gofmt` and `goimports`. The later will alphabetize and separate stdlib and 3rd party libs as seen here, but we've generally been using `gofmt` prior to this. If we want to make the switch to `goimports`, let's make that call, but until then, use `gofmt`.